### DISCUSSION TITLE
my github pages used github token,  i deploy github pages vercel  and visited get error: Failed to load resource: the server responded with a status of 401?

### DISCUSSION THREAD

**Discussion by labike** [vercel/community#4646] [Help]
### Summary

my github pages used github token, and i visited xxx.github.io is ok. i deploy vercel, and visited https://xxx-github-io.vercel.app/ get error: Failed to load resource: the server responded with a status of 401, how resolve the error and help me, thanks everyone!

### Steps to Reproduce

my github pages used github token, and i visited xxx.github.io is ok. i deploy vercel, and visited https://xxx-github-io.vercel.app/ get error: Failed to load resource: the server responded with a status of 401, how resolve the error and help me, thanks everyone!

**Comment by swarnava** [comment]
Hi,

Please share us a minimal repo so we can take a closer look

**Reply by labike** [reply]
https://github.com/labike/labike.github.io/tree/master
